Jenkinsfile

```diff
@@ -56,7 +56,7 @@ pipeline {
         stage('Tests') {
             steps {
                 configFileProvider([configFile(fileId: 'maven-settings-with-deploy-snapshot', variable: 'MAVEN_SETTINGS')]) {
-                    sh "$MAVEN_HOME/bin/mvn verify -B -V -e -s $MAVEN_SETTINGS -Dmaven.test.failure.ignore=true -Dansi.strip=true"
+                    sh "$MAVEN_HOME/bin/mvn verify -B -V -e -s $MAVEN_SETTINGS -Dmaven.test.failure.ignore=true -Dansi.strip=true -Dsurefire.rerunFailingTestsCount=5"
```

Comment:
Do you want this in master and PR builds? I thought you only wanted to be able to do it in your own builds.

Reply:
Yes. Because it will start reporting what are the flaky tests.

Reply:
How will that happen? Is there any documentation that you could link to in a comment here?

Reply:
I would like to use https://plugins.jenkins.io/flaky-test-handler but it is not working. We need to spend more time but the first goal of this PR is to remove `<disableXmlReport>false</disableXmlReport>` that is the first restriction from the surefire maven plugin